core/src/com/unciv/logic/automation/civilization/NextTurnAutomation.kt

```diff
@@ -147,14 +162,16 @@ object NextTurnAutomation {
                     value -= (20 - distance) / 4
             }
         }
-        else if (civInfo.wantsToFocusOn(Victory.Focus.CityStates)) {
+        if (civInfo.wantsToFocusOn(Victory.Focus.CityStates)) {
             value += 5  // Generally be friendly
         }
         if (civInfo.getHappiness() < 5 && cityState.cityStateFunctions.canProvideStat(Stat.Happiness)) {
             value += 10 - civInfo.getHappiness()
+            value += (civInfo.getPersonality()[PersonalityValue.Happiness]).toInt() - 5
         }
         if (civInfo.getHappiness() > 5 && cityState.cityStateFunctions.canProvideStat(Stat.Food)) {
             value += 5
+            value += (civInfo.getPersonality()[PersonalityValue.Food]).toInt() - 5
```

Comment:
Sounds like getPersonalityValue(personality) one liner, you have this 5 time

Reply:
I'm down for getting rid of these parenthesis, but now that I look at it, wouldn't using `.get` be better than that? Actually, on second look, I don't need the parenthesis in the first place